manifests/i/Insco/FaForever/0.0.1.23/Insco.FaForever.installer.yaml

```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS1.CRLF.7-4-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Insco.FaForever
+PackageVersion: 0.0.1.23
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+AppsAndFeaturesEntries:
+- DisplayVersion: 0.0.1
+UpgradeBehavior: install
+ReleaseDate: 2024-06-17
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/cyf/faforever/releases/download/v0.0.1+23/FaForever_0.0.1.23_x64_en-US.msi
+  InstallerSha256: 7D452D0552A4A891AB1BFBF69953E725E4FCF032665EA897C139BFE60F35F4BF
+  ProductCode: '{4588C71C-9148-41B7-8327-37676105D419}'
```

Comment:
These product codes don't look correct
```
--> Comparing ARP Entries

DisplayName                     DisplayVersion Publisher             ProductCode                            Scope
-----------                     -------------- ---------             -----------                            -----
FaForever                       0.0.1.23       kjxbyz                {05286340-8BD3-4C3F-B298-A5FCFF5A1655} Machine
```

Reply:
How are these being extracted from the Registry?

Reply:
How do I get the product code correctly?